scripts/run_pylint.py

```diff
@@ -72,7 +72,7 @@
 }
 TEST_RC_REPLACEMENTS = {
     'FORMAT': {
-        'max-module-lines': 1960,
+        'max-module-lines': 2000,
```

Comment:
1. We have a separate config for unit test files (you'll notice this is in a var. called `TEST_RC_REPLACEMENTS`)
1. This file is the biggest test file
1. I didn't want the refactor to be part of this PR (it would make it harder to review)
1. I'm happy to have it broken up, but don't have bandwidth